### PR TITLE
feat: logout button added

### PR DESCRIPTION
**Pull request by bhrg3se**

### CODE REVIEW

**Comment by bhrg3se** [comment]
We need a reference of the webview component to call [injectJavaScript](https://github.com/react-native-webview/react-native-webview/blob/master/docs/Guide.md#the-injectjavascript-method) method  (not to be confused with [injectedJavaScript](https://github.com/react-native-webview/react-native-webview/blob/master/docs/Guide.md#the-injectedjavascript-prop) props which runs js code only once in the beginning)

File: mobileapp/src/pages/Accounts/ScanProtect.tsx [hunk 1]
```diff
@@ -86,7 +86,16 @@ export const ScanAndProtect = (props: any) => {
             primaryColor={true}
           />
           {/* <ProgressBar style={styles.progress} progress={1} width={200} indeterminate={data? false:true} /> */}
-          <ScanResult result={result}/>
+          <ScanResult result={result} />
+          <Button
+            onPress={() => {
+              scannerRef.current?.injectJavaScript(
+                'alert(123);document.querySelector(\'a[href^="https://accounts.google.com/Logout"]\').click();true',
```
Comment (sshahcodes) [inline mobileapp/src/pages/Accounts/ScanProtect.tsx:93]: remove alert
```suggestion
                'document.querySelector(\'a[href^="https://accounts.google.com/Logout"]\').click();true',
```

File: mobileapp/src/pages/Accounts/ScanProtect.tsx [hunk 2]
```diff
@@ -86,7 +86,16 @@ export const ScanAndProtect = (props: any) => {
...
+              );
+            }}
+          >
+            LOGOUT!!!!
```
Comment (sshahcodes) [inline mobileapp/src/pages/Accounts/ScanProtect.tsx:97]:
```suggestion
            Sign out from this account
```

**Review by sshahcodes — CHANGES_REQUESTED** [review]
Make required changes

**Comment by sshahcodes** [comment]
> We need a reference of the webview component to call [injectJavaScript](https://github.com/react-native-webview/react-native-webview/blob/master/docs/Guide.md#the-injectjavascript-method) method (not to be confused with [injectedJavaScript](https://github.com/react-native-webview/react-native-webview/blob/master/docs/Guide.md#the-injectedjavascript-prop) props which runs js code only once in the beginning)

Okay, but why are we not using this in Fix functions?

**Comment by bhrg3se** [comment]
> Okay, but why are we not using this in Fix functions?

We probably should

**Review by sshahcodes — APPROVED** [review]

**Pull request merged**